sidecar/codec.go

```diff
@@ -0,0 +1,39 @@
+package sidecar
+
+import (
+	"bytes"
+	"encoding/base64"
+)
+
+const (
+	// defaultEncodeBufSize is the default buffer size we use when encoding
+	// a ticket.
+	defaultEncodeBufSize = 512
```

Comment:
Is this really needed?

Reply:
No, left over from a previous design. Removed.